torch/distributions/multinomial.py

```diff
@@ -2,7 +2,7 @@
 from torch._six import inf
 from torch.distributions.distribution import Distribution
 from torch.distributions import Categorical
-from numbers import Number
+import numbers
```

Comment:
seems like only `Integral` was used? should we just keep original format?
```suggestion
from numbers import Integral
```

Reply:
> seems like only `Integral` was used? should we just keep original format?

Nice catch. Will update.